src/react/useWatchTextFile.ts

```diff
@@ -1,115 +1,149 @@
 import React from "react";
-import { HandshakeStatus, WriteChangeArgs } from "src/types";
+import { WriteChange } from "src/types";
 import useReplit from "./useReplit";
 
+export enum Status {
```

Comment:
Can we move this enum to `types/index.ts` to ensure that it gets documented?

Reply:
Seems like we don't have a good docs setup if we have to do this. Should be able to document types exported from any file :(

Reply:
I'll move it for now